clients/venice-client/src/main/java/com/linkedin/venice/fastclient/meta/InstanceHealthMonitor.java

```diff
@@ -106,74 +162,45 @@ public ChainedCompletableFuture<Integer, Integer> trackHealthBasedOnRequestToIns
     TimeoutProcessor.TimeoutFuture timeoutFuture = null;
     if (transportFuture != null) {
       timeoutFuture = timeoutProcessor.schedule(
-          /** Using a special http status to indicate the leaked request */
+          /** Using a special http status to indicate the timed out request */
           () -> {
             transportFuture.completeExceptionally(new VeniceClientHttpException("Request timed out", SC_GONE));
+            // TODO: maybe remove this log as the logging frequency can be high
+            LOGGER.warn(
```

Comment:
We can use a `RedundantLogFilter` for this.

Reply:
Fixed